src/main/java/jenkins/plugins/logstash/persistence/ElasticSearchDao.java

```diff
@@ -192,4 +197,24 @@ private String getErrorMessage(CloseableHttpResponse response) {
       }
     }
   }
+
+  public static void main(String[] args) throws URISyntaxException, MalformedURLException
+  {
+    URI uri = (new URIBuilder("localhost:8000")).setPath("/logstash").build();
+
+    //System.out.println(uri.toURL());
```

Comment:
commented out code

actually it's weird FB isn't flagging this

Reply:
forgot to delete this main class